Index: libsquish-1.14/Doxyfile
===================================================================
--- libsquish-1.14.orig/Doxyfile
+++ libsquish-1.14/Doxyfile
@@ -190,7 +190,7 @@ PERL_PATH              = /usr/bin/perl
 #---------------------------------------------------------------------------
 CLASS_DIAGRAMS         = YES
 HIDE_UNDOC_RELATIONS   = YES
-HAVE_DOT               = NO
+HAVE_DOT               = YES
 CLASS_GRAPH            = YES
 COLLABORATION_GRAPH    = YES
 GROUP_GRAPHS           = YES
